prow/cmd/deck/main.go

```diff
@@ -41,6 +41,7 @@ import (
 var (
 	configPath   = flag.String("config-path", "/etc/config/config", "Path to config.yaml.")
 	buildCluster = flag.String("build-cluster", "", "Path to file containing a YAML-marshalled kube.Cluster object. If empty, uses the local cluster.")
+	tideURL      = flag.String("tide-url", "", "Path to tide. If empty, do not serve tide data.")
```

Comment:
`If empty, do not serve tide data.` is not enforced.

Reply:
Done.